Content.Server/NPC/HTN/PrimitiveTasks/Operators/PlaySoundOperator.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Robust.Server.Audio;
+using Robust.Shared.Audio;
+
+namespace Content.Server.NPC.HTN.PrimitiveTasks.Operators;
+
+public sealed partial class PlaySoundOperator : HTNOperator
+{
+    private AudioSystem _audio = default!;
+
+    [DataField(required: true)]
+    public SoundSpecifier? Sound;
+
+    public override void Initialize(IEntitySystemManager sysManager)
+    {
+        base.Initialize(sysManager);
+
+        _audio = IoCManager.Resolve<IEntitySystemManager>().GetEntitySystem<AudioSystem>();
+    }
```

Comment:
It's right there mate.

Reply:
Oh, for example I use other HTN opertator (Say), In the future I may also do HTN, I’ll have to fix that